velox/functions/prestosql/UuidFunctions.h

```diff
@@ -20,10 +20,47 @@
 
 #include "velox/functions/Macros.h"
 #include "velox/functions/Registerer.h"
+#include "velox/functions/prestosql/Comparisons.h"
 #include "velox/functions/prestosql/types/UuidType.h"
 
 namespace facebook::velox::functions {
 
+struct UuidCompareBase {
+  FOLLY_ALWAYS_INLINE int compare64(uint64_t x, uint64_t y)
```

Comment:
These can be static functions or free functions.

Let's also put this code into 'details' namespace.